lib/luanetfilter.c

```diff
@@ -89,19 +191,27 @@ static const lunatik_class_t luanetfilter_class = {
 	.sleep = false,
 };
 
-static int luanetfilter_xtable(lua_State *L)
+static void luanetfilter_release(void *private)
 {
-	lunatik_object_t *object;
-	luanetfilter_xtable_t *luaxtable;
-
-	object = lunatik_newobject(L, &luanetfilter_class, sizeof(luanetfilter_xtable_t));
-	luaxtable = (luanetfilter_xtable_t *)object->private;
+	if (match) {
```

Comment:
these globals (match and target) are temporary while we don't have the RCU table, right?

Reply:
Yes, exactly